tests/jobs/test_api.py

```diff
@@ -33,6 +33,27 @@ async def test_find_beta(archived, state, fake, snapshot, spawn_client):
         assert all(job["state"] == state for job in body["documents"])
 
 
+@pytest.mark.parametrize("job_filter", [None, "finished", "complete", "failed"])
+async def test_delete(job_filter, fake2, spawn_client, test_job, resp_is, snapshot):
+    client = await spawn_client(authorize=True, permissions=[Permission.remove_job])
+
+    user = await fake2.users.create()
+
+    test_job["user"] = {"id": user.id}
+
+    await client.db.jobs.insert_one(test_job)
+
+    url = "/jobs?"
+
+    if job_filter:
+        url += f"&filter={job_filter}"
```

Comment:
The `&` is only required when there is more than one path parameter.
```suggestion
    url = "/jobs"

    if job_filter:
        url += f"?filter={job_filter}"
```